src/lib/dexs/index.ts

```diff
@@ -14,24 +14,27 @@ import { Shell } from './shell';
 import { Cofix } from './cofix';
 
 const DEXS: { [dex: string]: typeof Adapter } = {
-  'uniswap': UniswapV1,
-  'uniswapv2': UniswapV2,
-  'sushiswap': UniswapV2,
-  'defiswap': UniswapV2,
-  'linkswap': UniswapV2,
-  'kyber': Kyber,
-  'bancor': Bancor,
-  'paraswappool': Zerox,
-  'zerox': Zerox,
-  'curve': Curve,
-  'curve3': Curve,
-  'aave': Aave,
-  'aave2': Aave,
-  'compound': Compound,
-  'weth': Weth,
-  'balancer': Balancer,
-  'shell': Shell,
-  'cofix': Cofix,
+  uniswap: UniswapV1,
+  uniswapv2: UniswapV2,
+  sushiswap: UniswapV2,
+  defiswap: UniswapV2,
+  linkswap: UniswapV2,
+  kyber: Kyber,
+  bancor: Bancor,
+  paraswappool: Zerox,
+  paraswappool2: Zerox,
+  paraswappool3: Zerox,
+  paraswappool4: Zerox,
```

Comment:
Can we not specify the individual paraswappool and just use a regex (similar to on in transaction-builder.ts). This will allow us to not modify the SDK for each market maker